cmd/kots/cli/root.go

```diff
@@ -21,8 +22,8 @@ func RootCmd() *cobra.Command {
 
 	cobra.OnInitialize(initConfig)
 
-	kubernetesConfigFlags = genericclioptions.NewConfigFlags(false)
-	kubernetesConfigFlags.AddFlags(cmd.PersistentFlags())
+	k8sutil.KubernetesConfigFlags = genericclioptions.NewConfigFlags(false)
```

Comment:
i think this would be better off in the `init` function in `k8sutil` package. and also since this is the only place `KubernetesConfigFlags` is used directly, it could be un-exported.